src/workerd/jsg/modules.h

```diff
@@ -181,6 +181,10 @@ class ModuleRegistry {
       // The EXTERNAL flag tells the compile operation to treat the content as an external
       // string wrapping an immutable buffer outside the V8 heap. When this flag is not set,
       // the content is copied into the V8 heap.
+      COMPILE_CACHE,
```

Comment:
You are using these enumerants as a bitfield, but you aren't explicitly specifying their numeric values. It just so happens to work with the flags you've specified so far, but if someone were to add another flag after this one without specifying the numeric value, it would be 3, which does not specify a unique bit.

Can you explicitly assign values like:

```
NONE = 0,
EXTERNAL = 1 << 0,
COMPILE_CACHE = 1 << 1
```

so it's more obvious these are bitfield values?

Reply:
Replacing the separate `EXTERNAL` and `COMPILE_CACHE` flags with a single `BUILTIN` flag makes the need to treat these as a bitfield obsolete.